articles/communication-services/quickstarts/telephony-sms/includes/send-sms-java.md

```diff
@@ -119,21 +117,33 @@ Instantiate an `SmsClient` with your connection string. The code below retrieves
 Add the following code to the `main` method:
 
 ```java
+// Your can find your endpoint and access key from your resource in the Azure Portal
+String endpoint = "https://<RESOURCE_NAME>.communication.azure.com";
+String accessKey = "SECRET";
+
 // Create an HttpClient builder of your choice and customize it
 HttpClient httpClient = new NettyAsyncHttpClientBuilder().build();
 
-CommunicationClientCredential credential = new CommunicationClientCredential(accessKey);
-
 // Configure and build a new SmsClient
 SmsClient client = new SmsClientBuilder()
     .endpoint(endpoint)
-    .credential(credential)
+    .accessKey(accessKey)
```

Comment:
This will not work. There is no accessKey property, you need to use the CommunicationClientCredential.

Reply:
@chrwhit  These doc changes were made in accordance to the changes in the this PR https://github.com/Azure/azure-sdk-for-java/pull/15917. As soon as the changes go in java, I will merge the change in the docs.